Doc/library/netrc.rst

```diff
@@ -18,9 +18,10 @@ the Unix :program:`ftp` program and other FTP clients.
 .. class:: netrc([file])
 
    A :class:`~netrc.netrc` instance or subclass instance encapsulates data from  a netrc
-   file.  The initialization argument, if present, specifies the file to parse.  If
-   no argument is given, the file :file:`.netrc` in the user's home directory --
-   as determined by :func:`os.path.expanduser` -- will be read.  Otherwise,
+   file. The initialization argument, if present, specifies the file to parse. If no
+   argument is given, it will look for the file path in the :envvar:`!NETRC` environment variable.
+   If that is not set, it defaults to reading the file :file:`.netrc` in the user's home
+   directory -- as determined by :func:`os.path.expanduser`. If the file cannot be found,
```

Comment:
```suggestion
   argument is given, it will look for the file path in the :envvar:`!NETRC` environment variable,
   before defaulting to reading the file :file:`.netrc` in the user's home
   directory -- as determined by :func:`os.path.expanduser`. If the file cannot be found,
```